File: chapter1_fundamentals/setting_swapping.py
def setting_swapping():
    # swapping value
    # change myName value with myNumber  and vice versa
    myNumber = 42
    myName = 'Ali'
    print "my number before ", myNumber
    print "my name before ", myName
    temp = myNumber
    myNumber = myName
    myName = temp

    print "My Name after", myName
    print "My Number after", myNumber


print(setting_swapping())

File: JavascriptPractice/setting_swapping.js
// Set myNumber to 42. set myName to your name. now swap myNumber into myName and vice versa


var myNumber = 42;
var myName = "Ali";
console.log("original myNumber", myNumber);
console.log("original myName", myName);
var temp = myNumber;

myNumber = myName;
myName = temp;

console.log("my number after switch", myNumber);
console.log("my name after switch",  myName);
